docs/secure/Idp_integration.md

```diff
@@ -0,0 +1,143 @@
+# Percona Everest IdP integration
+
+[Identity Provider (IdP)](https://www.okta.com/identity-101/why-your-company-needs-an-identity-provider/) integration connects applications and services with an external identity provider for your organization. This enables centralized authentication and authorization management, improving security and simplifying user access. By leveraging IdP integration, you can ensure that users are securely authenticated and authorized to access various applications and services across your organization.
+
+Percona Everest uses [OpenID Connect](https://auth0.com/docs/authenticate/protocols/openid-connect-protocol)(OIDC) Protocol to integrate with external Identity Providers (IdP).
+
+
+## Configure OIDC on the providers's side
+
+- **PKCE**: When setting up the Provider side, it's important to configure an application specifically for [PKCE](https://auth0.com/docs/get-started/authentication-and-authorization-flow#authorization-code-flow-with-proof-key-for-code-exchange-pkce-) (Proof Key for Code Exchange) authorization.
+
+    !!! info "Important"
+        The login redirect URL should point to `<EVEREST_URL>/login-callback`, and the logout redirect URL to `<EVEREST_URL>`.
+
+- **https based**: IdP providers often require a secure connection (https). Therefore, the provider might require your `<EVEREST_URL>` to be based on https.
+
+- **Issuer URL**: The created OIDC Application will have a `client ID` field, which can be used for the Everest configuration. The issuer URL can be found in your provider's Authorization Servers list.
+
+    ??? example "Example: OKTA"
+
+        1. Sign in to your Okta organization as a user with administrative privileges.
+        
+        2. In the **Admin** Console, go to **Applications → Applications** and click **Create App Integration**.
+
+        3. On the **Create a new app integration page**, set the following:
+        
+            - Sign-in method - **OIDC - OpenID Connect**
+            - **Application type** - **Signle-Page Application**, and click **Next**.
+
+            ![!image](../images/OKTA_admin_console.png)
+
+        4. Set the following fields:
+
+            a. App integration name - any value
+
+            b. Sign-in redirect URIs - `<EVEREST_URL>/login-callback`
+
+            c. Sign-out redirect URIs - `<EVEREST_URL>`
+
+            d. Click **Save**.
+
+            e. Copy the `clientID` of the created app.
+
+            f. Navigate to **Security → API → Authorization Servers** and copy the `issuerURL` you’d like to use for the Everest authorization. 
+
+            !!! note "Note"
+                Okta allows the use of HTTP for development purposes and in cases where the Admin explicitly permits it.
+
+            ![!image](../images/sso_aap_integration.png)
+
+
+
+## Configure OIDC on Percona Everest side
+
+
+You can configure OIDC via Percona Everest CLI:
+
+    everestctl settings oidc configure --issuer-url=http://url.com --client-id=id2342
```

Comment:
Just for consideration.  
There are two options how we can run the config - using flags (as described here) or using wizard. 

The flags option's output looks like:
```
$ everestctl settings oidc configure --issuer-url=<your-provider-url> --client-id=<your-app-client-id>
2024-06-18T11:06:18Z    info    oidc/configure.go:110   OIDC provider configured, restarting Everest..
2024-06-18T11:06:33Z    info    oidc/configure.go:117   OIDC has been configured successfully
```

The wizard option looks like:
```
$ everestctl settings oidc configure                                     
? Enter issuer URL <your-provider-url>
? Enter client ID <your-app-client-id>
2024-06-18T11:05:15Z    info    oidc/configure.go:110   OIDC provider configured, restarting Everest..
2024-06-18T11:05:30Z    info    oidc/configure.go:117   OIDC has been configured successfully
```

Reply:
Fixed